src/commonMain/kotlin/com/akelius/university/statistic/core/dto/SlideshowScore.kt

```diff
@@ -2,5 +2,6 @@ package com.akelius.university.statistic.core.dto
 
 data class SlideshowScore(
     val slideScores: List<SlideScore>,
-    val isQuiz: Boolean
+    val isQuiz: Boolean,
+    val type: String = "DEFAULT"
```

Comment:
If type is used only for quiz multiple, maybe rename it to quizMultipleType? Or it will be used for other calculation types?

Reply:
It can be used to transfer any info on slideshow, in the future if we need to calculate score for some different type, it would be set here.